vue/src/VariableSelect/VariableSelect.vue

```diff
@@ -0,0 +1,211 @@
+<!--
+  Matomo - free/libre analytics platform
+  @link https://matomo.org
+  @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+-->
+
+<template>
+  <div class="tagManagerManageSelect tagManagerVariableSelect">
+    <ActivityIndicator :loading="isLoading" />
+    <div class="tableActionBar">
+      <a
+        class="createNewVariable"
+        v-show="!isLoading"
+        @click="createVariable()"
+      >
+        <span class="icon-add" />
+        {{ translate('TagManager_CreateNewX', translate('TagManager_Variable')) }}
+      </a>
+    </div>
+    <ul class="collection with-header">
+      <li class="collection-header">
+        <h4>{{ translate('TagManager_CustomVariables') }}</h4>
+      </li>
+      <li
+        class="collection-item"
+        @click="selectVariable(variable)"
+        v-for="(variable, index) in containerVariables"
+        :key="index"
+      >
+        <span class="title">{{ variable.name }} (Type: {{ variable.type }})</span>
```

Comment:
We could maybe directly use the translation here, even though it was missing before ;-)

```suggestion
        <span class="title">{{ variable.name }} ({{ translate('TagManager_Type') }}: {{ variable.type }})</span>
```